bears/r/FormatRBear.py

```diff
@@ -68,24 +68,15 @@ def create_arguments(filename, file, config_file,
                 1
             ```
         """
-        rcode = ('library(formatR);'
-                 'formatR::tidy_source('
-                 'source="' + escape(filename, '"\\') + '",'
-                 'comment={r_keep_comments},'
-                 'blank={r_keep_blank_lines},'
-                 'arrow={r_use_arrows},'
-                 'brace.newline={r_braces_on_next_line},'
-                 'indent='
-                 '{tab_width}'.format(tab_width=tab_width,
-                                      r_keep_comments=_map_to_r_bool(
-                                          r_keep_comments),
-                                      r_keep_blank_lines=_map_to_r_bool(
-                                          r_keep_blank_lines),
-                                      r_use_arrows=_map_to_r_bool(
-                                          r_use_arrows),
-                                      r_braces_on_next_line=_map_to_r_bool(
-                                          r_braces_on_next_line)))
-        # Disable r_max_expression_length if it is equal to 0
-        rcode += ('' if r_max_expression_length == 0
-                  else ',width.cutoff=' + str(r_max_expression_length)) + ')'
+        options = {'source="' + escape(filename, '"\\') + '"',
+                   'blank=' + _map_to_r_bool(r_keep_blank_lines),
```

Comment:
okay :)

2016-06-21 12:23 UTC+01:00, AbdealiJK <notifications@github.com>:
>> -                 'brace.newline={r_braces_on_next_line},'
>> -                 'indent='
>> -                 '{tab_width}'.format(tab_width=tab_width,
>> -                                      r_keep_comments=_map_to_r_bool(
>> -                                          r_keep_comments),
>> -                                      r_keep_blank_lines=_map_to_r_bool(
>> -                                          r_keep_blank_lines),
>> -                                      r_use_arrows=_map_to_r_bool(
>> -                                          r_use_arrows),
>> -
>> r_braces_on_next_line=_map_to_r_bool(
>> -                                          r_braces_on_next_line)))
>> -        # Disable r_max_expression_length if it is equal to 0
>> -        rcode += ('' if r_max_expression_length == 0
>> -                  else ',width.cutoff=' + str(r_max_expression_length)) +
>> ')'
>> +        options = {'source="' + escape(filename, '"\\') + '"',
>> +                   'blank=' + _map_to_r_bool(r_keep_blank_lines),
>
> Just `r_bool` is probably a simpler name compared to `_map_to_r_bool` ...
> although it's not in the scope of this PR.
> Could you make an issue for it ?
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/coala-bears/pull/525/files/3dbc2ae9d11ff726d82950116bc1543e121752b9#r67850512